blueocean-web/src/main/java/io/jenkins/blueocean/ResourceTag.java

```diff
@@ -0,0 +1,68 @@
+package io.jenkins.blueocean;
+
+import org.apache.commons.jelly.JellyTagException;
+import org.apache.commons.jelly.TagSupport;
+import org.apache.commons.jelly.XMLOutput;
+import org.jvnet.maven.jellydoc.annotation.NoContent;
+import org.kohsuke.stapler.framework.adjunct.AdjunctManager;
+import org.kohsuke.stapler.framework.adjunct.AdjunctsInPage;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletContext;
+
+@NoContent
+public class ResourceTag extends TagSupport {
```

Comment:
@tfennelly started replicating all the stapler adjunct tags here but ran out of time. Looks reasonably straight forward to copy this jelly tag lib into BO and migrate them to use `Resource Manager`.